app/workers/delete_provided_access_tokens_worker.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DeleteProvidedAccessTokensWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :low
+
+  def perform
```

Comment:
We could also make `now` an argument of `perform` so eventual job failures and corresponding retries wouldn't act on different sets of records, but maybe not worth it since, in the end, all expired provided access tokens should be purged anyway.

Reply:
OK I get it now :) Why the `long_expired` method